packages/blockchain-link/src/workers/solana/index.ts

```diff
@@ -51,10 +84,12 @@ const getAllSignatures = async (
 
     const limit = 100;
     while (keepFetching) {
-        const signaturesInfos = await api.getSignaturesForAddress(new PublicKey(descriptor), {
-            before: lastSignature?.signature,
-            limit,
-        });
+        const signaturesInfos = await api.rpc
+            .getSignaturesForAddress(address(descriptor), {
+                before: lastSignature?.signature,
+                limit,
+            })
+            .send();
```

Comment:
Here is one example of a callsite where you could introduce an `AbortSignal` if ever you wanted to make these RPC-related operations cancellable.

```ts
    .send({ abortSignal });
```